### DISCUSSION TITLE
Export data with API

### DISCUSSION THREAD

**Discussion by supermarin1** [apache/superset#13231] [Q&A / Help]
Is there the way to export data charts/dashboards based on with REST API?

Tried `/chart/export/`, `/database/export/`, `/dashboard/export/` and have no luck. Or I understand wrong meaning of `export`? 

Please, advise.

**Comment by gidinetapp** [comment]
you can get all of them with:
`/api/v1/assets/export/`

**Reply by supermarin1** [reply]
no luck again 
<img width="1708" alt="image" src="https://github.com/apache/superset/assets/24755510/669e98dc-9a5c-466b-8936-13fb5cc381b0">

**Reply by SaadRevSureAi** [reply]
How I can export the specific dashboard
